docs/account-management/connecting-github-and-semaphore.md

```diff
@@ -0,0 +1,100 @@
+
+It is advised that you use GitHub App as the method of connecting your repositories because of the two main advantages:
+
+- **More granular permission** - Using GitHub App allows you to give access to each repository individually.
+- **No OAuth token owner** - GitHub app doesn't rely on the personal access token of the original Semaphore project owner, making the user offboarding easier. 
+
+### Connecting a repository via GitHub App
+In order to connect the repository via [Semaphore GitHub App](https://github.com/apps/semaphore-ci-cd) three conditions need to be met:
+1. You are a collaborator on the repository.
+2. You installed the GitHub App on the GitHub organization/account that the repository belongs to.
+3. You gave the GitHub App access either to all repositories or the one you want to connect to.
+
+If the conditions above are not met, you will see the empty repository list when trying to create the project:
+
+![GH App - Empty list](.images/ghapp_zero.png)
+
+**To give access to more repositories:**
+
+**Step 1.** Click on the "Give access to more repositories"  
+**Step 2.** Select GitHub account/organization on which you want to install the app  
+**Step 3.** Select "All repositories" or pick individual ones you want to give access to.   
+**Step 4.** You will be returned to the repository list in the Semaphore and repositories should appear.   
+
+You can always edit and update the access permissions of each installed GitHub App. 
+
+!!! info "Installing the GitHub App on GitHub organization"
+    You might not be able to install the GitHub app on the GitHub organizations that you're not the owner of. In such cases following the steps above will only **request** the installation from the organization owner. 
+    The repositories will not be available until the organization owner approves the installation in GitHub. 
+    Once you request the installation, the GitHub organization owner will receive the email request for approval from GitHub.
+
+### Connecting a repository via a personal access token
+If you want to connect the repository via [Semaphore OAuth app](https://github.com/settings/connections/applications/328c742132e5407abd7d) following conditions have to be met:
+
+1. You are the collaborator on the repository.
+2. You gave Semaphore OAuth app access to your repositories (public or public & private).
+3. You approved the Semaphore OAuth app GitHub organization (applies only if the repository is in GitHub organization and not the personal account).
+
+Depending on how many conditions above are met, the repository list can either: 
+- Contain all the repositories you have access to.
+- Contain only the repositories from your personal account.
+- Contain no repositories and prompting you to give Semaphore OAuth app access:
+![OAuth permissions](.images/oauth_permissions.png)
+
+**To connect your first project:**  
```

Comment:
**In order to connect your first project, follow these steps**
1.
2.
3.

Reply:
Def. sounds better. Updated.